### PR TITLE
Release notes fixups

### LINKED ISSUE #17444: Release notes tools are confusing
Right now, we can use `release-notes/release-notes.go` in [kubernetes/contrib](https://github.com/kubernetes/contrib) to do release notes for alpha releases, (since it properly munges PRs on master,) but must use `build/make-release-notes.sh` for official versions, because it properly follows cherry-picks back to their original PRs, (if they're formatted right, which they aren't always).

These tools should be merged and made more reasonable.

### PR DESCRIPTION
**Pull request by ikehz**
Mark a release as stable when we announce it, and stop using `cherry_pick_list.sh`.

FYI @roberthbailey 

Fixes #17444.

### CODE REVIEW

File: docs/devel/releasing.md [hunk 1]
```diff
@@ -170,39 +170,33 @@ and follow the instructions.
...
-    release.
-- Official release:
-  - From your clone of upstream/master, run `./hack/cherry_pick_list.sh ${RELEASE_VERSION}`
-    to get the release notes for the patch release you just created. Feel free
-    to prune anything internal, but typically for patch releases we tend to
-    include everything in the release notes.
-  - If this is a first official release (vX.Y.0), look through the release
-    notes for all of the alpha releases since the last cycle, and include
-    anything important in release notes.
+email to the team with the template.  Figure out what the PR numbers for this
```
Comment (roberthbailey) [inline docs/devel/releasing.md]: We should set up a mailing list for this that is visible outside of google. 

@sarahnovotny @jbeda @countspongebob

**Comment by roberthbailey** [comment]
LGTM.

**Comment by fgrzadkowski** [comment]
@k8s-bot unit test this please

File: docs/devel/releasing.md (hunk 1 above)
Comment (ihmccreery) [inline docs/devel/releasing.md]: Filed #18662.

**Comment by wojtek-t** [comment]
@ihmccreery - it is failing with the following error:

FAIL: changes needed but not made due to --verify
/home/travis/gopath/src/github.com/kubernetes/kubernetes/docs/ is out of date. Please run hack/update-generated-docs.sh
!!! Error in hack/verify-generated-docs.sh:28

Can you please fix it?

**Comment by ihmccreery** [comment]
Just a lint issue; reapplying LGTM.

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please

**Pull request merged**